advanced/Exercise_5_git-rebase.md

```diff
@@ -95,19 +95,27 @@ Add and commit this change.
 git commit -am "Add Apero to day 2"
 ```
 
-Now you are ready to incorporate the changes you made on your feature branch into the *main* branch. This time we'll do it with a rebase instead of a merge.
+Now you are ready to incorporate the changes you made on your feature branch into the `main` branch. This time we'll do it with a rebase instead of a merge.
 
 ```plaintext
 git rebase main rebase_feature
 ```
 
-Examine the repository status and log. Git has switched us to the *rebase_feature* branch and played the commit from the *main* branch here, without creating a merge commit.
+Examine the repository status and log. Git has switched us to the `rebase_feature` branch and played the commit from the `main` branch here, without creating a merge commit.
 
-Now, switch to the *main* branch and sync it with the *rebase_feature* branch.
+Now, switch to the `main` branch and sync it with the `rebase_feature` branch.
 
 ```plaintext
 git switch main
 git merge rebase_feature
 ```
 
-Examine the repository status and log again. Even though we did a merge command, by doing the rebase beforehand, we have created a situation where the merge can be done using the fast-forward technique, and therefore no merge commit was required. This is particularly advantageous in actively developed codes because it allows the history of the *main* branch to remain clean and free of unnecessary merge commits.
+Examine the repository status and log again. Even though we did a merge command, by doing the rebase beforehand, we have created a situation where the merge can be done using the fast-forward technique, and therefore no merge commit was required. This is particularly advantageous in actively developed codes because it allows the history of the `main` branch to remain clean and free of unnecessary merge commits.
+
+Don't forget to delete the `rebase_feature` branch after this merge.
+
+```plaintext
+git branch -d rebase_feature
+```
+
+To visualise the difference, take a look at the Git log graph (remember what we learned in [Exercise 1](./Exercise_1_git-log-diff-show.md#use-git-log-to-have-a-look-at-the-repository))
```

Comment:
I think we use American English (visuali**z**e) for most of the rest. Can be considered as another consistency check.